README_CN.md

```diff
@@ -106,20 +106,16 @@ CeresDBClient 是 CeresDB 的高性能 Java 版客户端。CeresDB 是定位为
 ## 初始化 CeresDB Client
 ```java
 // CeresDB options
-final CeresDBOptions opts = CeresDBOptions.newBuilder("127.0.0.1", 8831) //
-        .tenant("test", "sub_test", "test_token") // tenant info
-        // maximum retry times when write fails
-        // (only some error codes will be retried, such as the routing table failure)
-        .writeMaxRetries(1)
-        // maximum retry times when read fails
-        // (only some error codes will be retried, such as the routing table failure)
-        .readMaxRetries(1)
+final CeresDBOptions opts = CeresDBOptions.newBuilder("127.0.0.1", 8831, DIRECT) // 默认 gprc 端口号，DIRECT 模式
+        .tenant("public", "sub_test", "test_token") // 租户信息
```

Comment:
Remove sub tenant config, it is useless now.

Reply:
doing

Reply:
fixed, use `database` now